scamp/scamp-3.c

```diff
@@ -1244,9 +1245,9 @@ void proc_100hz (uint a1, uint a2)
                 }
             }
         }
-      else if (netinit_tick_counter >= 3)
+      else if (netinit_biff_tick_counter >= 20)
```

Comment:
I have reverted this to 3 and testing is showing no difference.  Do the other changes seem OK?